table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -316,26 +320,33 @@ public UdbTable getTable(String tableName) throws IOException {
 
       // Glue does not provide column statistic information
       List<ColumnStatisticsInfo> columnStatisticsData = new ArrayList<>();
+      Map<String, String> tableParameters = table.getParameters() == null
+          ? new HashMap<>() : table.getParameters();
```

Comment:
If `tableParameters` does not need to be mutable, then we can just use `Collections.emptyMap()`

Reply:
Changed.